upgrades/upgrade_testing_framework/steps/step_start_source_cluster.py

```diff
@@ -0,0 +1,29 @@
+from upgrade_testing_framework.core.framework_step import FrameworkStep
+from upgrade_testing_framework.cluster_management.cluster import Cluster, ClusterNotStartedInTimeException
+
+class StartSourceCluster(FrameworkStep):
+    """
+    This step starts the source cluster and ensures it is healthy
+    """
+
+    def _run(self):
+        # Get the state we need
+        source_docker_image = self._get_state_value("test_config")["source_docker_image"]
+        docker_client = self.state.docker_client
+
+        # Begin the step body
+        self.logger.info("Creating source cluster...")
+        cluster = Cluster("source-cluster", 3, docker_client, source_docker_image)
```

Comment:
There's a lot of configuration we could be passing in here from the test configuration, I am assuming we just haven't got to that point yet but will in the future

Reply:
Yeah, I'm fleshing that out right now in my code for the next task.